src/builder.rs

```diff
@@ -976,6 +999,20 @@ fn build_with_store_internal(
 				Arc::clone(&node_metrics),
 			))
 		},
+		Some(ChainDataSourceConfig::Electrum { server_url, sync_config }) => {
+			let sync_config = sync_config.unwrap_or(ElectrumSyncConfig::default());
+			Arc::new(ChainSource::new_electrum(
```

Comment:
(from commit msg, `Runtime` available)
>This might change in the future, but for now we do need this workaround.

What would be required for this potential change?

Reply:
I think I misinterpreted 'this' in the commit message then. I thought it referred to getting the runtime available in the builder. Could perhaps be made explicit. Or could also be clear enough for people with enough understanding of the code and async.